src/main/resources/assets/lambdynlights/lang/de_de.json

```
@@ -4,16 +4,16 @@
   "lambdynlights.menu.canvas.1": "Mit Canvas ist es empfohlen, die dynamische Entitäten-Beleuchtung auszuschalten!",
   "lambdynlights.menu.canvas.2": "Dynamische Beleuchtung mit Canvas ist zur Zeit sehr Leistungsbedürftig!",
```

```suggestion
  "lambdynlights.menu.canvas.2": "Dynamische Beleuchtung mit Canvas ist zur Zeit sehr leistungsbedürftig!",
```
Small spelling correction.